RevitPythonShell/DefaultConfig/init.py

```diff
@@ -60,32 +60,61 @@ def __init__(self, uiApplication):
         import RevitLookup
         self.RevitLookup = RevitLookup
         # See note in CollectorExt.cs in the RevitLookup source:
-        self.RevitLookup.Snoop.CollectorExts.CollectorExt.m_app = uiApplication
-        self.revit = uiApplication
-
-    def lookup(self, element):
+        #self.RevitLookup.Snoop.CollectorExts.CollectorExt.m_app = uiApplication
+        #self.revit = uiApplication
+    def IsInstalled(self):
         if not self.RevitLookup:
             print('RevitLookup not installed. Visit https://github.com/jeremytammik/RevitLookup to install.')
-            return
-        if isinstance(element, int):
-            element = self.revit.ActiveUIDocument.Document.GetElement(ElementId(element))
-        if isinstance(element, ElementId):
-            element = self.revit.ActiveUIDocument.Document.GetElement(element)
-        if isinstance(element, list):
-            elementSet = ElementSet()
-            for e in element:
-                elementSet.Insert(e)
-            element = elementSet
-        form = self.RevitLookup.Snoop.Forms.Objects(element)
-        form.ShowDialog()
+            return False
+        return True
+
+    def SnoopCurrentSelection(self):
+        if self.IsInstalled()== True:
+            form = self.RevitLookup.Views.ObjectsView()
+            form.SnoopAndShow(self.RevitLookup.Core.Selector.SnoopCurrentSelection)
+    def SnoopElement(self,element):
+        if self.IsInstalled()== True:
+            if element is None:
+                print("element null object, Please input element to snoop")
+                return
+            if isinstance(element, int):
+                element = doc.GetElement(ElementId(element))
+            if isinstance(element, ElementId):
+                element = doc.GetElement(element)
+            if isinstance(element, list):
+                elementSet = ElementSet()
+                for e in element:
+                    elementSet.Insert(e)
+                element = elementSet
+            form = self.RevitLookup.Views.ObjectsView(element)
+            self.RevitLookup.Core.ModelessWindowFactory.Show(form)
+    def SnoopActiveView():
```

Comment:
this is nitpicking, but there are some formatting guides for python. methods start with a lower-case, and it's customary to add an empty line in between.

Reply:
Hi @daren-thomas , origin project Revitlookup use upper-case, whether we can follow it ?
Example : self.RevitLookup.Core.Selector.SnoopCurrentSelection with SnoopCurrentSelection us upper case ?

Reply:
I updated all code standard use same with project revitlookup, if you have any problem please to me know reason, thanks you.